### DISCUSSION TITLE
한개의 병원에 한 testmode만 실행할 수 있나요?

### DISCUSSION THREAD

**Discussion by seokwonchoisnu** [changdoc/naver-vaccine-macro#77] [Q&A]
한개의 병원에서만 2000ms로 1시간동안 자동예약할 수 있는건가요? 더많은 병원에서 동시에 새로고침을 눌러서 작동시키면 차단되나요?

**Comment by OneTop4458** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/42
